Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -559,7 +640,7 @@ def fetch_incidents(client: Client, max_results: Optional[str], last_run: dict,
     alerts = alerts_response_data.get('data')
     alerts = arrange_alerts_by_incident_type(alerts)
     incidents, fetch_start_time, last_fetch_id = alerts_to_incidents_and_fetch_start_from(
-        alerts, fetch_start_time, last_run)
+        alerts, str(fetch_start_time), last_run)
```

Comment:
since you changed here already, let's add a log line with the next_run info for future bugs.